docker/build.Dockerfile

```diff
@@ -45,54 +20,21 @@ ARG RAIDENVERSION=master
 # This is a "hack" to automatically invalidate the cache in case there are new commits
 ADD https://api.github.com/repos/${REPO}/commits/${RAIDENVERSION} /dev/null
 
-# clone raiden
-RUN mkdir -p /apps && \
-    git clone https://github.com/${REPO}.git /apps/raiden
-
-WORKDIR /apps/raiden
-
-# install requirements (replacing all --editable requirements)
-RUN git checkout $RAIDENVERSION && \
-    sed -s 's/^-e //' requirements.txt > _requirements.txt && \
-    /raiden.AppDir/usr/bin/pip install -r _requirements.txt
-
-# build & install raiden with contracts, webui and smoketest
-RUN echo "recursive-include raiden/ui/web/dist *" >> MANIFEST.in && \
-    USER=root \
-    NPM_CONFIG_PREFIX=/tmp/node_modules \
-    NODE_PATH=/tmp/node_modules \
-    PATH=/tmp/node-v8.2.1-linux-x64/bin:/raiden.AppDir/usr/bin:$PATH \
-    /usr/bin/env python setup.py compile_contracts compile_webui build install && \
-    find /raiden.AppDir/ -iname '*.pyo' -exec rm {} \; && \
-    rm /raiden.AppDir/usr/lib/libpython*.a
-
-WORKDIR /
-
-# add .desktop file
-ADD raiden.desktop /raiden.AppDir/raiden.desktop
-RUN cd /apps/raiden && \
-    VERSIONSTRING=$(/raiden.AppDir/usr/bin/raiden version --short) && \
-    sed -s -i "s/XXVERSIONXX/$VERSIONSTRING/" /raiden.AppDir/raiden.desktop
-# add icon
-ADD raiden.svg /raiden.AppDir/raiden.svg
-
-RUN curl -L -o functions.sh "https://github.com/probonopd/AppImages/raw/master/functions.sh"
+# clone raiden repo + install dependencies
+RUN git clone https://github.com/raiden-network/raiden
```

Comment:
This call should use `${REPO}` instead of hardcoding the repo name.
Also it's currently always cloning `master` and not the requested `${RAIDENVERSION}`